Protocol.md

```diff
@@ -613,7 +613,7 @@ The general format for an extension frame is given below.
                           Depends on Extended Type...
 ```
 
-* __Frame Type__: (16) 0xFFFF for Extension Header.
+* __Frame Type__: (16) 0xFFFF
```

Comment:
Actually, that was a different place. The other place was fixed. Not sure how this was still wrong :-) Anyhow ... fixed now.